docs/includes/apiargs-function-add_logger-param.yaml

```diff
@@ -0,0 +1,11 @@
+arg_name: param
+name: $logger
+type: Psr\\Log\\LoggerInterface
```

Comment:
For "reasons", we can't use a URL here. I tried:

```
`Psr\Log\LoggerInterface <https://www.php-fig.org/psr/psr-3/#3-psrlogloggerinterface>`__
```

Giza complains about a leading backtick character. Text roles (e.g. `:php:`) work, but we don't have one for the PSR website.

I _was_ able to specify a URL when also using multi-line string syntax (e.g. `type: |`), but that appends a trailing " or" to the output. I expect there's something in the build system that supports multiple types on separate lines -- which we've never used since we typically write "array|object".

Anyway, the PSR website is linked from the main doc page so this isn't a huge problem.